environments/dev-env/README.md

```diff
@@ -0,0 +1,515 @@
+# KCP-Edge User Kit Environment
+
+## Required Packages:
+   - ko: https://ko.build/install/ 
+   - gcc
+   - docker 
+   - jq
+   - make
+   - go (version expected 1.19)
```

Comment:
These (excepting docker or a drop-in replacement such as podman) are not needed by users, right?

Reply:
no

Reply:
I hate it when I ask a question in a negative way.

Are all of those pre-reqs needed by users?